Bucketeer/Sources/Internal/Model/MetricsEventData.swift

```diff
@@ -1,5 +1,17 @@
 import Foundation
 
+protocol MetricsEventDataProps {
+    var apiId: ApiId { get }
+    var labels: [String: String] { get }
+    var protobufType: String? { get }
```

Comment:
Shouldn't this be required?
I don't remember a case that the `protobufType` will be nil.

Reply:
This protocol follows up with the same current implementation. You could see on each  `metrics_event` 
Yes, I think we should make it non-null as our current test case force it non-null when checking.

Reply:
Understood.
Thanks for fixing it.